src/sections/Meshmap/index.js

```diff
@@ -101,9 +103,19 @@ const Meshmap = () => {
                       form: "meshmap",
                     }}
                     onSubmit={values => {
-                      setMemberFormOne(values);
-                      setStepNumber(1);
-                      nextStep();
+                      if(google.value||github.value||twitter.value||linkedin.value) {
+                        setMemberFormOne(values);
+                        setStepNumber(1);
+                        nextStep();
+                      }
+                      else {
+                        setValidateAccounts(true);
+                        firstname.value=values.firstname;
+                        email.value=values.email;
+                        lastname.value=values.lastname;
+                        occupation.value=values.occupation;
+                        org.value=values.org;
```

Comment:
Can you please give some suggestions how I can change the role value beacause it's saying undefined when I am changing it in the same manner as above?

Reply:
Resolved this by tracking the initialvalue of role.